lib/pages/settings.dart

```diff
@@ -226,6 +226,18 @@ class _SettingsPageState extends State<SettingsPage> {
                             height16,
                             colourDropdown,
                             height16,
+                            Row(
```

Comment:
This should be directly below the `themeDropdown` as the two settings are related.